### PR TITLE
replace usage of $this->content for $content to prevent type mismatch

### PR DESCRIPTION
**Pull request by fmarquesto**
While using a newer version of Symfony, the attribute $content defines its type to string and this causes a type issue when returning `$this->getOriginalContent()` as this method returns `mixed`

### CODE REVIEW

**Comment by rdeutz** [comment]
There is already a PR to fix this issue #47

**Comment by fmarquesto** [comment]
I know @rdeutz but I can't use third-party forks on my projects,
when your gets approved and merged I will close this one 💪

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
         "mockery/mockery": "~1.0",
         "phpunit/phpunit": "^9.0|^10.0",
         "squizlabs/php_codesniffer": "~2.0",
-        "php-open-source-saver/jwt-auth": "^1.4"
+        "php-open-source-saver/jwt-auth": "^2.2"
```
Comment (specialtactics) [inline composer.json:36]: Can you please make this "^1.4 | ^2.2"

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
         "email": "jason.lewis1991@gmail.com"
     }],
     "require": {
-        "php": "^8.0",
+        "php": "^8.0|^8.1|^8.2",
```
Comment (specialtactics) [inline composer.json:16]: This isn't necessary, "^8..0" already allows for every 8.x version.

**Comment by specialtactics** [comment]
Hey @fmarquesto I left some minor feedback, and also check the failing styleci check (very minor change needed).

**Comment by specialtactics** [comment]
Also @fmarquesto, I think @rdeut is correct, this mostly duplicates that PR, the one dep you updated, just address my feedback and we can merge that by itself.

File: composer.json (hunk 1 above)
Comment (fmarquesto) [reply composer.json]: done ✅

File: composer.json (hunk 2 above)
Comment (fmarquesto) [reply composer.json]: and done ✅

**Comment by fmarquesto** [comment]
@specialtactics I've addressed both issue, plus the style.
Thanks for your feedback!

**Review by specialtactics — APPROVED** [review]

**Comment by specialtactics** [comment]
Looks good thank you @fmarquesto going to merge and tag as 4.2.2

**Pull request merged**